docs/lib/gst-player-sections.txt

```diff
@@ -82,6 +82,11 @@ GstPlayerSubtitleInfo
 
 gst_player_media_info_get_uri
 gst_player_media_info_get_duration
+gst_player_media_info_get_title
+gst_player_media_info_get_container_format
+gst_player_media_info_is_seekable
+gst_player_media_info_get_cover_image
```

Comment:
how does gst_player_media_info_get_image_sample () sounds ? this name tells exactly what we are returning :)